lte/gateway/python/integ_tests/s1aptests/test_attach_detach_multiple_secondary_pdn.py

```diff
@@ -25,9 +25,38 @@ def test_multiple_seconday_pdn_conn_req(self):
         """ Attach a single UE + add 2 PDN Connections + disconnect """
         num_pdns = 2
         bearer_ids = []
-        apn = ["ims", "internet"]
-        self._s1ap_wrapper.configUEDevice(1)
-
+        num_ue = 1
```

Comment:
Can you fix the typo in the function name seconday should be secondary in "test_multiple_seconday_pdn_conn_req"